### DISCUSSION TITLE
bfffb6bfdf358eb713zknj

### DISCUSSION THREAD

**Discussion by bbtinoz** [milo526/homebridge-tuya-web#554] [General]
Hi All

I am using the TUYA Web Plugin - but cant find the device Name with ID bfffb6bfdf358eb713zknj

Its a Bonaire Heater.

Any help is greatly appreciated. 

Tony

**Comment by wizardofozzie** [comment]
Which heater out of interest? Split system?

**Comment by bbtinoz** [comment]
This one

Bonaire Portable Electric Heater... https://www.amazon.com.au/dp/B0BY1V1XHR?ref=ppx_pop_mob_ap_share

Thanks

Tony

Sent from my iPhone

On 7 Jul 2023, at 14:29, WizardOfOzzie ***@***.***> wrote:

﻿

Which heater out of interest? Split system?
